addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -254,22 +250,37 @@ registerModel({
          * @private
          */
         _prepareAttachmentsLoading() {
-            this._isPreparingAttachmentsLoading = true;
-            this._attachmentsLoaderTimeout = this.env.browser.setTimeout(() => {
-                this.update({ isShowingAttachmentsLoading: true });
-                this._isPreparingAttachmentsLoading = false;
-            }, this.messaging.loadingBaseDelayDuration);
+            this.update({
+                _attachmentsLoaderTimeout: this.env.browser.setTimeout(
+                    () => {
+                        this.update({
+                            isShowingAttachmentsLoading: true,
+                            _isPreparingAttachmentsLoading: clear(),
+                        });
+                    },
```

Comment:
general in whole diff: let's take the opportunity to create methods on the model rather than nested anonymousgeneral in whole diff: let's take the opportunity to create methods on the model rather than nested anonymous

Reply:
Made change for all but the 2 record methods `_debounce` in `RtcCallViewer` and `RtcSession`, because they call function `f` passed as param so we cannot just pass the function with no context.